release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -189,6 +189,13 @@ A new `--node-selector` argument provides a way to identify which nodes you are
 
 For more information, see xref:../support/gathering-cluster-data.adoc#support-network-trace-methods_gathering-cluster-data[Network trace methods] and xref:../support/gathering-cluster-data.adoc#support-collecting-host-network-trace_gathering-cluster-data[Collecting a host network trace].
 
+[id="ocp-4-10-openshift-sdn-netpol-egress-policies"]
+==== OpenShift SDN cluster network provider network policy support for egress policies and ipBlock except
```

Comment:
The title is quite long; not sure if it can stand on its own by starting with just, "Network policy support for..."?

Reply:
@bobfuru, true, but don't want to confuse _OVN-Kubernetes cluster network provider_ users.